doc/org_export.md

```diff
@@ -21,3 +26,47 @@ in-built publishing or ox-hugo -- use the following snippet to add a
 
 (add-hook 'org-export-before-processing-hook 'my/org-export-preprocessor)
 ```
+
+
+### Backlinks and Contents
+
+This would insert both backlinks and the contents, just like the org-roam buffer. This might be especially useful if you host a web version of your personal knowledgebase and want to browse the files along with the backlinks from mobile devices.
+
+```emacs-lisp
+  (defun my/org-roam--backlinks-list-with-content (file)
+    (with-temp-buffer
+      (if-let* ((backlinks (org-roam--get-backlinks file))
+                (grouped-backlinks (--group-by (nth 0 it) backlinks)))
+          (progn
+            (insert (format "\n\n* %d Backlinks\n"
+                            (length backlinks)))
+            (dolist (group grouped-backlinks)
+              (let ((file-from (car group))
+                    (bls (cdr group)))
+                (insert (format "** [[file:%s][%s]]\n"
+                                file-from
+                                (org-roam--get-title-or-slug file-from)))
+                (dolist (backlink bls)
+                  (pcase-let ((`(,file-from _ ,props) backlink))
+                    (insert (propertize
+                             (s-trim (s-replace "\n" " "
+                                                (plist-get props :content)))
+                             'help-echo "mouse-1: visit backlinked note"
```

Comment:
This function could use some cleanup. You don't need to propertize the inserted string since it's only for export purposes.

Reply:
Tried to clean it up. I am new to Lisp and keeping track of parentheses is difficult :disappointed: Had to commit twice because messed up few parens here and there.